development-docs/TESTING.md

```diff
@@ -148,29 +148,36 @@ The following table shows currently used tags:
 
 | Name               | Description                                                                        |
 | :----------------: | :--------------------------------------------------------------------------------: |
-| travis             | Marks tests executed on Travis                                                     |
 | acceptance         | Acceptance tests, which guarantee, that basic functionality of Strimzi is working. |
 | regression         | Regression tests, which contains all non-flaky tests.                              |
 | upgrade            | Upgrade tests for specific versions of the Strimzi.                                |
+| travis             | Marks tests executed on Travis                                                     |
 | flaky              | Execute all flaky tests (tests, which are failing from time to time)               |
+| scalability        | Execute scalability tests                                                          |
```

Comment:
WHat's difference between this and `scale`?

Reply:
I was thinking about it, I split it to `scalability` -> test where big amount of users/topics are created, `scale` -> resource is scaled up/down.

But my first idea was to keep only `scalability`. Do you want to remove the `scale` tag?

Reply:
> I was thinking about it, I split it to `scalability` -> test where big amount of users/topics are created, `scale` -> resource is scaled up/down.
> 
> But my first idea was to keep only `scalability`. Do you want to remove the `scale` tag?

I would say, that both your cases fall into `scalability` tag group :)

Reply:
Okay :D I wasn't sure, thanks